service/controller/clusterapi/v27/ebs/ebs.go

```diff
@@ -64,7 +65,24 @@ func (e *EBS) DeleteVolume(ctx context.Context, volumeID string) error {
 
 		return nil
 	}
-	b := backoff.NewExponential(30*time.Second, 5*time.Second)
+
+	var b cenkaltibackoff.BackOff
+	{
+		b := &cenkaltibackoff.ExponentialBackOff{
+			// If EBS cannot be deleted on first try, it's probably still in use so it
+			// makes sense to give some time for unmount/detach logic to complete
+			// before retrying.
+			InitialInterval:     5 * time.Second,
```

Comment:
`backoff.DefaultInitialInterval` is 500ms. I think this should be at least 5s. Maybe even 10s.